benchexec/tools/fusebmc.py

```diff
@@ -5,41 +5,50 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import benchexec.tools.esbmc as esbmc
-from benchexec.tools.template import ToolNotFoundException
+import os
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
 
-
-class Tool(esbmc.Tool):
+class Tool(benchexec.tools.template.BaseTool2):
```

Comment:
Please apply the formatting of our code formatter `black` (two blank lines before the class, no trailing whitespace).